splunk_topology/test_splunk_topology.py

```diff
@@ -1,18 +1,22 @@
 # stdlib
 import json
 import os
+import mock
```

Comment:
I see no tests for splunk telemetry, we should have those aswell

Reply:
Yeah i couldn't see the test written for the `splunk_telemetry_base` in sts-agent. So i didn't write it.